### PR TITLE
[ACA-4238] - added aspect list dialog on context menu action

### PR DESCRIPTION
**Pull request by VitoAlbano**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [X] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
No aspect list dialog is displayed

**What is the new behaviour?**
Aspect list dialog is displayed as context action

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by eromano — APPROVED** [review]

**Comment by adinapitul** [comment]
I see many different terminology used here: change aspect, edit aspect, aspect list, manage aspects.... Can we have some consistency?

File: e2e/suites/actions-available/files-folders/test-data.ts [hunk 1]
```diff
@@ -55,10 +56,21 @@ const fileSharedFavLockedContextMenu = [
   'Copy',
   'Delete',
   'Manage Versions',
+  'Edit Aspects',
```
Comment (adinapitul) [inline e2e/suites/actions-available/files-folders/test-data.ts:59]: I'm pretty sure this is not ok. User should not be able to Edit Aspects if the file is locked

**Review by adinapitul — CHANGES_REQUESTED** [review]
Same for all locked files expected actions

File: e2e/suites/actions-available/files-folders/test-data.ts [hunk 2]
```diff
@@ -928,25 +928,25 @@ export const folderFav2 = {
 // ---- multiple selection ---
 
 // TODO: raise issue to remove 'Permissions'
-const multipleSelContextMenu = ['Download', 'Favorite', 'Move', 'Copy', 'Delete', 'Permissions'];
+const multipleSelContextMenu = ['Download', 'Favorite', 'Move', 'Copy', 'Delete', 'Edit Aspects', 'Permissions'];
```
Comment (adinapitul) [inline e2e/suites/actions-available/files-folders/test-data.ts:921]: I'd say Edit Aspects should NOT be displayed for multiple selection

**Review by adinapitul — CHANGES_REQUESTED** [review]

File: e2e/suites/actions-available/special-permissions/test-data-permissions.ts [hunk 3]
```diff
@@ -29,9 +29,9 @@ export const random = Utils.random();
 
 // ----- files -----
 
-const consumerContextMenu = ['Share', 'Download', 'View', 'Favorite', 'Copy', 'Manage Versions'];
-const consumerFavContextMenu = ['Share', 'Download', 'View', 'Remove Favorite', 'Copy', 'Manage Versions'];
-const consumerSharedContextMenu = ['Shared Link Settings', 'Download', 'View', 'Favorite', 'Copy', 'Manage Versions'];
+const consumerContextMenu = ['Share', 'Download', 'View', 'Favorite', 'Copy', 'Edit Aspects', 'Manage Versions'];
```
Comment (adinapitul) [inline e2e/suites/actions-available/special-permissions/test-data-permissions.ts:32]: Consumer and contributor should not see Edit Aspects.
Please revert this and check the docs.

**Review by adinapitul — CHANGES_REQUESTED** [review]

**Review by adinapitul — APPROVED** [review]

**Pull request merged**